CandidateExercise/ConfigurationExercise/ConfigValue.cs

```diff
@@ -8,9 +13,55 @@ public enum ValueType
         ValueString
     }
 
-    public struct ConfigValue
+    public class ConfigValue
     {
         public string Value { get; set; }
         public ValueType ValueType { get; set; }
-    }
+
+        private static readonly Dictionary<ValueType, Type> ValueTypeMap = new Dictionary<ValueType, Type>
```

Comment:
Maybe refactor this static dictionary outside of the ConfigValue class (so that you don't have several duplicate copies of the list for each ConfigValue created)?

I'm not 100% sure how a static list functions in an object class, but I would guess it creates a memory location for each instance of the ConfigValue class.

Reply:
I thought they worked on a one-instance-on-the-Type basis.

I agree it should go somewhere else though. Honestly I'd prefer to revert ConfigValue to a struct  and have it just function as a bag of state.